Cargo.toml

```diff
@@ -41,7 +41,7 @@ serde = { version = "1.0.103", default-features = false, optional = true }
 
 # You likely only want to enable these if you explicitly do not want to use "std", otherwise enable
 # the respective -std feature e.g., hashes-std
-hashes = { package = "bitcoin_hashes", version = ">= 0.12, <= 0.14", default-features = false, optional = true }
+hashes = { package = "bitcoin_hashes", version = ">= 0.12, < 0.15", default-features = false, optional = true }
```

Comment:
I wonder if Clippy would accept a lint to say that `< [major rev]` is likely to be wrong.

Reply:
https://github.com/rust-lang/rust-clippy/issues/13362 let's see.